inst/rmd/h/navigation-1.1/codefolding.js

```diff
@@ -45,7 +45,9 @@ window.initializeCodeFolding = function(show) {
     buttonCol.append(showCodeButton);
     buttonRow.append(buttonCol);
 
-    div.before(buttonRow);
+    var sourceCodeBlock = $('div.sourceCode');
```

Comment:
Will this also work with Pandoc 1.9.x? i.e. Does Pandoc 1.9 also generate `div.sourceCode`?

Reply:
This is part of the test I need to make. Yes pandoc 1.9.x have a `div.sourceCode` but the css applied to those part have changed. 

So yes this line work and the button is added above without issue, but the margin trick does not work. Will change that. 

Also the main fix 
```css
pre.sourceCode  {
  color: inherit;
  background-color: transparent;
}
```
does not work with pandoc 1.9 because in this pandoc version the background is applied to `pre.sourceCode` so it erase it... - 
if we want something that work with old and new, I need to find another one...  :confused:

Reply:
ok - there is not `div.sourceCode` inserted if we use `highlight = 'default'`, whatever the version of pandoc. I need to deal with that.